op-scim-bridge/values.yaml

```diff
@@ -6,10 +6,13 @@ scim:
   version: "{{ .Chart.AppVersion }}"
   # credentialsVolume configures the SCIM bridge to access the credentials via a volume
   credentialsVolume:
+    # enables the `credentialsVolume`. Defaults to `false`. Either `credentialsSecret` or `cedentialsVolume` must be specified, but not both
+    enabled: false
```

Comment:
This helm chart is the base upon which our marketplace applications are built. As they do the 'one-click' installation method which writes the value to the volume, we will need `credentialsVolume` to be default and `credentialsSecret` to be the alternate. 

For customers wishing to leverage the credentials secret directly, swapping the enabled flags themselves before deployment should be doable.

Reply:
These can be changed, but I would highly encourage you to change the marketplace entries to follow the secrets best practices. At the very least it would be helpful to call out that people should enable using secrets in the docs somewhere when they do the deploy themselves.